src/Uno.UI.Composition/Composition/Compositor.skia.cs

```diff
@@ -81,56 +81,46 @@ internal void Render(SKSurface surface)
 
 		internal void RenderVisual(SKSurface surface, Visual visual)
 		{
-			if (visual.Opacity != 0 && visual.IsVisible)
+			if (visual is { Opacity: 0 } or { IsVisible: false })
```

Comment:
@Youssef1313 
`¬(A ∧ B) ≡ (¬A ∨ ¬B)` / `¬(A ∨ B) ≡ (¬A ∧ ¬B)` :-)

Reply:
@carldebilly Yeah, I mean I won't use pattern matching :)
I don't see improved readability with `is` and `or` and braces compared with `visual.Opacity == 0 || !visual.IsVisible`. Also the latter doesn't have unnecessary null checks.

Reply:
I personally find it more readable with pattern matching, especially since it's only "positive" conditions. It indeed adds a null-check, but it will only make the code more robust, the cost of it is negligible.